rest-client/src/main/java/org/thingsboard/rest/client/RestClient.java

```diff
@@ -306,16 +305,24 @@ public void clearAlarm(AlarmId alarmId) {
     }
 
     public PageData<AlarmInfo> getAlarms(EntityId entityId, AlarmSearchStatus searchStatus, AlarmStatus status, TimePageLink pageLink, Boolean fetchOriginator) {
+        String urlSecondPart = "/api/alarm/{entityType}/{entityId}?fetchOriginator={fetchOriginator}&";
```

Comment:
Please remove the '&' from the end of this part. Use 

```java
        if(searchStatus != null) {
            params.put("searchStatus", searchStatus.name());
            urlSecondPart += "&searchStatus={searchStatus}";
        }
```

instead of 

```java
        if(searchStatus != null) {
            params.put("searchStatus", searchStatus.name());
            urlSecondPart += "searchStatus={searchStatus}&";
        }
```